src/LoadableModule.h

```diff
@@ -9,26 +9,26 @@
 #ifndef SQUID_SRC_LOADABLEMODULE_H
 #define SQUID_SRC_LOADABLEMODULE_H
 
-#include "SquidString.h"
+#include "sbuf/SBuf.h"
 
 // wrapper for dlopen(3), libltdl, and friends
 class LoadableModule
 {
 public:
-    LoadableModule(const String &aName);
+    LoadableModule(const SBuf &aName);
     ~LoadableModule();           // unloads if loaded
 
     bool loaded() const;
-    const String &name() const { return theName; }
-    const String &error() const { return theError; }
+    const SBuf &name() const { return theName; }
+    const SBuf &error() const { return theError; }
```

Comment:
I do not insist on this, but we can apply AAA here as well:

```suggestion
    const auto &name() const { return theName; }
    const auto &error() const { return theError; }
```